src/components/OSCALControlParamLegend.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { styled, useTheme } from "@mui/material/styles";
+import { Box } from "@mui/material";
+
+export default function OSCALControlParamLegend() {
+  const LegendLabel = styled("div")(
```

Comment:
I think you're right about us falling back on using the `<div>` tag a lot, and maybe to the point where we're overusing them. From our discussion last week, there seems to be a somewhat loose hierarchy of using container-like HTML tags. `<div>` is a container for block content, while `<p>` is a paragraph container. Due to the legend content being more block-like, I choose to use a `<div>`. Although, I could also see a `<span>` working in this case, because the content sits in-line, but these are generally used for marking up text.

Reply:
Fair point, I guess I was mostly referring to some of these `<div>` tags recreating something already provided by `React`. I did not know if there was already a more descriptive component we could use in some places.

Reply:
I think for now a styled `<div>` is a decent option for the `LegendLabel`. I have swapped the surrounding `<div>` for a fragment. We could return to this later if we find a better approach.